modules/telco-ran-ref-design-features.adoc

```diff
@@ -26,27 +26,25 @@ a|You can now use custom CRs alongside the reference configuration CRs provided
 
 * link:https://docs.openshift.com/container-platform/4.15/scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.html#ztp-adding-new-content-to-gitops-ztp_ztp-advanced-policy-config[Adding custom content to the {ztp} pipeline]
 
-//CNF-7078
-//|Intel Westport Channel e810 NIC as PTP Grandmaster clock
-//a|You can use the Intel Westport Channel E810-XXVDA4T as a GNSS-sourced grandmaster clock.
-//The NIC is automatically configured by the PTP Operator with the E810 hardware plugin.
-//This feature is scheduled for an upcoming 4.14 z-stream release.
+//CNF-8035
+|Using custom node labels in the `SiteConfig` CR with {ztp}
+a|You can now use the `nodeLabels` field in the `SiteConfig` CR to create custom roles for nodes in managed clusters.
 
-//* link:https://docs.openshift.com/container-platform/4.14/networking/using-ptp.html#configuring-linuxptp-services-as-grandmaster-clock_using-ptp[Configuring linuxptp services as a grandmaster clock]
+* link:https://docs.openshift.com/container-platform/4.15/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.html#ztp-sno-siteconfig-config-reference_ztp-deploying-far-edge-sites[{sno-caps} SiteConfig CR installation reference]
 
-//CNF-6527
-//|PTP Operator hardware specific functionality plugin
-//a|A new E810 NIC hardware plugin is now available in the PTP Operator.
-//You can use the E810 plugin to configure the NIC directly.
-//This feature is scheduled for an upcoming 4.14 z-stream release.
+//CNF-7078
+|Intel Westport Channel e810 NIC as PTP Grandmaster clock
+a|You can use the Intel Westport Channel E810-XXVDA4T as a GNSS-sourced grandmaster clock.
+The NIC is automatically configured by the PTP Operator with the E810 hardware plugin.
```

Comment:
This should probably be marked as tech-preview unless it has since been made GA in 4.14